source/documentation/deploying_services/mysql.md

```diff
@@ -132,7 +132,7 @@ To create a service and bind it to your app:
 
 ### Accessing MySQL from your app
 
-Your app must make a [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) connection to the MySQL service. Most libraries use TLS by default.
+Your app must make a [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) connection to the MySQL service. Some libraries use TLS by default, but others will need to be explicitly configured.
```

Comment:
For reviewer: do you think it's worth saying that the CA certificates are available in `/etc/ssl/certs`?